src/components/shared/ManagedTree.js

```diff
@@ -143,6 +143,10 @@ class ManagedTree extends Component<Props, State> {
   }
 
   focusItem = (item: Item) => {
+    const isPopupShown = document.querySelector("#contextmenu-mask.show");
+    if (isPopupShown !== null && typeof item === "undefined") {
```

Comment:
Not really sure what the state of `isPopupShown` is. In case it's just null or true we shouldn't really need a tripple equal, if on the other hand it can be `false`, `true` or `null` and we need to differentiate between `null` and `false` we do actually need the === here. Again, this is before looking into the code, so it's a general comment.

Checking against the string type of "undefined" we can omit the === here and replace it with a == since this equation never needs to be coerced. (or in other words: since `typeof` always returns a string and we compare it to a string `"undefined"` we don't need the additional type conversion that would result from ===)

Reply:
Oh I just use `===` out of habit. From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector), it's either `null` or an object. So it makes sense in both cases just to use `==`..